src/dp_transformers/dp_utils.py

```diff
@@ -92,6 +92,16 @@ def __call__(self, examples: List[Union[List[int], torch.Tensor, Dict[str, torch
         return batch
 
 
+class DataCollatorForPrivateSeq2Seq(DataCollatorForSeq2Seq):

```

Comment:
This is essentially a trivial wrapper class at the moment (nothing additional needed here). However, it might make sense to keep it as we add different `Seq2Seq` models.

Please share your thoughts.

Reply:
I have no opinion on this, maybe @wulu473 does?

Reply:
I would remove it and worry about creating a collator if we actually need it. The fewer lines of code the better.